app/controllers/content/index.js

```diff
@@ -82,6 +91,19 @@ export default Ember.Controller.extend(Analytics, {
         // True if the current user has admin permissions for the node that contains the preprint
         return (this.get('node.currentUserPermissions') || []).includes(permissions.ADMIN);
     }),
+
+    userIsContrib: Ember.computed('authors.[]', 'isAdmin', function() {
```

Comment:
Could this use/depend on `currentUser.currentUserId` instead of loading the user in `init`?